src/søknader/barnetilsyn/steg/6-barnepass/Barnepass.test.ts

```diff
@@ -0,0 +1,512 @@
+    ).toBeInTheDocument();
+
+    await klikkRadioknapp(
+      'Hvorfor trenger GÅEN PC pass?',
+      'Jeg må være borte fra hjemmet i lengre perioder på grunn av jobb',
+      screen,
+      user
+    );
+    expect(
+      screen.queryByText(
+        'Du må dokumentere behovet med uttalelse fra lege, spesialist eller annet helsepersonell.'
+      )
+    ).not.toBeInTheDocument();
+    expect(
+      screen.getByText(
+        'For at fraværet skal anses som mer langvarig enn vanlig, må det overstige 10 timer per dag'
+      )
+    ).toBeInTheDocument();
+    expect(screen.getByText('Du må dokumentere arbeidstiden din')).toBeInTheDocument();
+    expect(
+      screen.getByRole('group', { name: 'Hva slags barnepassordning har GÅEN PC?' })
+    ).toBeInTheDocument();
+
+    await klikkRadioknapp(
+      'Hvorfor trenger GÅEN PC pass?',
+      'Jeg jobber turnus eller skift, og jobber på tider utenom vanlig arbeidstid',
+      screen,
+      user
+    );
+    expect(
+      screen.queryByText(
+        'For at fraværet skal anses som mer langvarig enn vanlig, må det overstige 10 timer per dag'
+      )
+    ).not.toBeInTheDocument();
+    expect(screen.getByText('Du må dokumentere arbeidstiden din')).toBeInTheDocument();
+    expect(
+      screen.getByRole('group', { name: 'Hva slags barnepassordning har GÅEN PC?' })
+    ).toBeInTheDocument();
+
+    expect(
+      screen.getByText('Du må svare på alle spørsmålene før du kan gå videre til neste steg')
+    ).toBeInTheDocument();
+    expect(screen.queryByRole('button', { name: 'Neste' })).not.toBeInTheDocument();
+    expect(screen.getByRole('button', { name: 'Tilbake' })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: 'Avbryt' })).toBeInTheDocument();
+  });
+
+  test('Kan navigere seg til neste', async () => {
+    mockMellomlagretSøknadBarnetilsyn('barnetilsyn', '/barnetilsyn/barnepass');
+    const { screen, user } = await navigerTilStegBarnetilsyn();
+
+    await klikkRadioknapp(
+      'Hva slags barnepassordning har GÅEN PC?',
+      'Barnehage, SFO eller liknende',
+      screen,
+      user
+    );
+    await skrivFritekst('Navn på barnepassordningen', 'Barnehage', screen, user);
+    await skrivFritekst('Startdato', '01.06.2025', screen, user);
+    await skrivFritekst('Sluttdato', '01.12.2025', screen, user);
+    await skrivFritekst('Beløp pr måned (ikke inkludert kost)', '150', screen, user);
+    await klikkRadioknapp(
+      'Søker du om stønad til barnetilsyn fra en bestemt måned?',
+      'Nei, Nav kan vurdere fra hvilken måned jeg har rett til stønad',
+      screen,
+      user
+    );
+    await user.click(screen.getByRole('button', { name: 'Neste' }));
+
+    expect(screen.getByRole('heading', { level: 2, name: 'Oppsummering' })).toBeInTheDocument();
+  });
+});
+
+//Barn som er over 4.klassetrinn-test trengs. Se gift, 2 barn for eksempeltekst. Er både ekstra hjelpetekst og et ekstra spørsmål
```

Comment:
Burde det lages et Favro-kort i stedet for en kommentar i koden på den ekstra testen?